src/transformers/convert_graph_to_onnx.py

```diff
@@ -54,6 +55,9 @@ def __init__(self):
         self.add_argument(
             "--use-external-format", action="store_true", help="Allow exporting model >= than 2Gb",
         )
+        self.add_argument(
+            "--optimize", action="store_true", help="Let ONNX Runtime apply all the optimisations on the graph"
```

Comment:
For models that are not quantized, those optimizations can be done by ONNX Runtime automatically during creating session, so it is not needed to done it explicitly. Optimized model is less portable since some optimizations is device specified -- model optimized on CPU might not work on GPU, and vice versa.

Optimization is needed for quantization right now since model cannot be optimized after quantization (quantization will change graph).

So there are two options:
(1) Remove --optimize option, and apply optimization before quantization implicitly.
(2) Add a warning that optimized model might not run in the other type of device (CPU/GPU) or older version of ONNX Runtime.

Reply:
@tianleiwu Thanks for the detailed description here, it makes a lot of sense.

I think from a user perspective option (1) is the easiest way to approach such a quantization mechanism without asking too much input from the user.

Still, when using the `--quantize` argument we'll be saving one additional model (i.e. the optimized one). I think we need to print a message in the console to explain the user about the portability concern from using this `-optimized` model.

Reply:
@mfuntowicz, I agree that #1 sounds better for now. Onnx Runtime will print some warning during saving optimized model. You can have a test to see whether the warning could show up in the console.

Reply:
Optimization is now done automatically when `--optimize` is provided.

Reply:
looks like --optimize doesn't need any more.

Reply:
I totally forgot to remove the cli args. Thanks for catching @yufenglee 👍